sendReports.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+: ${MYSQL_HOST=openmrs-mysql-db}
+: ${MYSQL_ROOT_PASSWORD=secret_password}
+
+export MYSQL_HOST
+export MYSQL_ROOT_PASSWORD
+
+USERNAME=admin;
+POST_TEMPLATE=$(cat /postTemplate.json)
+
+processFile() {
+   REPORT_CONTENT=$(envsubst "`printf '${%s} ' $(bash -c "compgen -A variable")`"  < $1)
+   REPORT_CONTENT="${REPORT_CONTENT//\"/\\\"}"
+   REPORT_NAME=$(echo $2 | sed 's/\([A-Z][^A-Z]\)/ \1/g')
```

Comment:
Is this regex exclude small letter?

Reply:
This regex change camel case to the sentence. Example "sampleReport" will change to "sample Report".

Reply:
OK, it's fine for a set of filenames that we've defined before.